pkg/kubelego/watch.go

```diff
@@ -88,13 +151,24 @@ func (kl *KubeLego) WatchEvents() {
 			oldIng.ResourceVersion = ""
 			upIng.ResourceVersion = ""
 
-			if !reflect.DeepEqual(oldIng, upIng) {
```

Comment:
This would have played a large part of the flood of API requests I guess. If the status and meta were changing often.

Reply:
Yes exactly! 😬